test/e2e/scale_test.go

```diff
@@ -24,7 +24,26 @@ import (
 	"k8s.io/utils/pointer"
 )
 
-var _ = Describe("When scale testing using in-memory provider [Scale]", func() {
+var _ = Describe("When testing the machinery for scale testing using in-memory provider", func() {
```

Comment:
This is a duplicate in terms of spec, but the point is to have a test with no-test so we're testing e2e for the in-memory provider to ensure it's up and running.


This test should run as part of e2e-full while leaving the [Scale] tag for future work around scale testing.